package.json

```diff
@@ -22,7 +22,8 @@
     "start:watch": "nodemon --signal SIGTERM --watch src --ext js,jsx --exec babel-node src/server/index.js",
     "prestart:prod-sample": "npm run build:prod-sample",
     "start:prod-sample": "docker-compose -f ./prod-sample/docker-compose.yml up --abort-on-container-exit",
-    "test:lint": "eslint --ext js,jsx,md,snap .",
+    "test:lockfile": "lockfile-lint -p package-lock.json -t npm -a npm -o https: -c -i",
+    "test:lint": "eslint --ext js,jsx,md,snap . && npm run test:lockfile",
```

Comment:
can you add this to `test` instead of `test:lint`?

Reply:
Will do that.

Reply:
updated in 04ca91f